dftimewolf/lib/state.py

```diff
@@ -389,8 +394,11 @@ def _RunModuleProcessThreaded(
             f'Launching {module.name}.Process thread with {str(c)} from '
             f'{c.metadata.get(interface.METADATA_KEY_SOURCE_MODULE, "Unknown")}'
         )
+        time_start = time.time()
         futures.append(executor.submit(module.Process, c))
-
+        total_time = utils.CalculateRunTime(time_start)
+        TELEMETRY.LogTelemetry(
+          'total_time', str(total_time), 'core', self.recipe.get('name', 'no_name'))
```

Comment:
I'd argue that the key here should rather be something like `total_time_<module_name>` so that we can differentiate the entries per module.

Alternatively - you can also change the `'core'` parameter to be `core-<module_name>`.

Reply:
Same comment for the changes in L365